app/javascript/layouts/Layout.tsx

```diff
@@ -183,11 +183,7 @@ const Layout = (props: LayoutProps) => {
             mainContentId={"main-content"}
           />
 
-          <main
-            data-testid="main-content-test-id"
-            id="main-content"
-            className="md:overflow-x-hidden"
```

Comment:
For reviewer: I removed this and did some regression testing and it doesn't seem to affect anything! Let me know if you find something otherwise